portal/assignment.py

```diff
@@ -292,3 +281,72 @@ def assignment_grades():
 
         return render_template('layouts/teacher/assignments/assignment-grades.html', assignment=assignment, assignment_name=assignment_name)
     return redirect(url_for('teacher.home'))
+@bp.route('grades/grade-book', methods=('GET', 'POST'))
```

Comment:
There is also already an endpoint at '/assignments/gradebook' that seems to do something similar, maybe pick a different name and add docstrings to clarify the purpose of each endpoint?